### ISSUE TITLE
SSR: useLayoutEffect does nothing on the server

### ISSUE THREAD

**Issue by GriffinSauce** [uploadcare/react-widget#45]
Our react app does SSR, server render returns this error:

```
Warning: useLayoutEffect does nothing on the server, because its effect cannot be encoded into the server renderer's output format. This will lead to a mismatch between the initial, non-hydrated UI and the intended UI. To avoid this, useLayoutEffect should only be used in components that render exclusively on the client. See https://fb.me/react-uselayouteffect-ssr for common fixes.
    in Config
    in Fragment
    in ForwardRef (at Uploader.js:76)
    ...noise removed...
```

It's called here: https://github.com/uploadcare/react-widget/blob/311b6e41be835c9c0bbe8f8beacfa82b77d3df5e/src/index.js#L13

Is there a way to sidestep this?

**Comment by jeetiss** [comment]
@GriffinSauce released the new version v1.0.2, feel free to upgrade!

**Comment by GriffinSauce** [comment]
Late response but thank you for the super-quick fix! 🙌
